src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -117,6 +170,22 @@ def validate(self, attrs: dict[str, Any]) -> dict[str, Any]:
                     code="invalid",
                 )
 
+        # Make sure we have a variable-eigenshap mapping
+        if variables_properties := attrs.get("variables_properties"):
```

Comment:
The value of the zaakeigenschap can only be a string, so maybe we also need to check that the variables selected don't have complex types.